src/pretix/base/models/event.py

```diff
@@ -409,7 +410,12 @@ class Event(EventMixin, LoggedModel):
     )
     seating_plan = models.ForeignKey('SeatingPlan', on_delete=models.PROTECT, null=True, blank=True,
                                      related_name='events')
-
+    sales_channels = MultiStringField(
+        verbose_name=_('Restrict to specific sales channels'),
+        help_text=_('Only sell tickets for this event on the following sales channels.'),
+        default=['web'],
+        blank=True,
```

Comment:
Hmm, do we really want to allow "empty"? It adds another case for the support case "my event does not show up"…